src/models/__tests__/profile.test.ts

```diff
@@ -355,6 +356,25 @@ describe("createProfile", () => {
         newProfileWithNotificationsContentType
       )();
 
+      expect(containerMock.items.create).toBeCalledTimes(1);
+      expect(containerMock.items.create).toBeCalledWith(
+        expect.objectContaining({
+          kind: newProfileWithNotificationsContentType.kind,
+          fiscalCode: newProfileWithNotificationsContentType.fiscalCode,
+          id: generateVersionedModelId<Profile, "fiscalCode">(
+            newProfileWithNotificationsContentType.fiscalCode,
+            0 as NonNegativeInteger
+          ),
+          version: 0,
+          // the decoder here sets the field to UNSET for undefined values
+          pushNotificationsContentType:
+            newProfileWithNotificationsContentType.pushNotificationsContentType !==
+            undefined
+              ? newProfileWithNotificationsContentType.pushNotificationsContentType
+              : "UNSET"
```

Comment:
Why this login in here? IMHO is the same of `expectedPushNotificationsContentType`

Reply:
yes it is unnecessary as you said, sorry.